docs/infrastructure/infra-ui.asciidoc

```diff
@@ -1,68 +1,111 @@
 
 [float]
 [[infra-metric]]
-=== Start at a high-level by selecting the metric
+=== Select the metric to view
 
-This filter helps you start focusing on potential problem areas that may need
-further investigation. You'll see metrics that are most relevant for hosts or
-the container you selected.
+Select the metric to view from the *Metric* dropdown list.
+The available metrics are those that are most relevant for the high-level view you selected.
 
 [float]
 [[infra-group]]
-=== Group components 
+=== Group components
+
+Select the way you want to group the infrastructure components from the *Group By* dropdown list.
+The available options are specific to your physical, virtual, or container-based infrastructure.
+Examples of grouping options include *Availability Zone*, *Machine Type*, *Project ID*, and *Cloud Provider* for hosts, and *Namespace* and *Node* for Kubernetes.
+
+[float]
+[[infra-search]]
+=== Use the power of search
 
-The *Group By* selector offers grouping options that are native and specific for
-your physical, virtual, and container-based infrastructure.
-Examples include Availability Zone, Machine Type, Project ID, and Cloud Provider
-for Hosts, and Namespace and Node for Kubernetes.
+Use the search bar to perform ad hoc or structured searches using {kibana-ref}/kuery-query.html[Kibana Query Language] which features autocomplete and a simple, easy to use syntax.
+For example, enter `host.hostname : "host1"` to see only the information for `host1`.
 
 [float]
 [[infra-date]]
-=== Specify the date range
+=== Specify the time and date
 
-Use the time selector to focus on a specific timeframe.
+Click the time selector to choose the timeframe for the metrics.
+The values shown are the values for the last minute at the specified time and date.
 
 [float]
 [[infra-refresh]]
-=== Auto-refresh or pause
+=== Auto-refresh metrics
 
-Set auto-refresh to keep up-to-date information coming in, or stop
-refreshing to focus on historical data without new distractions.
+Select *Auto-refresh* to keep up-to-date metrics information coming in, or *Stop refreshing* to focus on historical data without new distractions.
 
 [float]
 [[infra-configure-source]]
-=== Adapt to your metric source
+=== Configure the data to use for your metrics
+
+The default source configuration for metrics is specified in the {kibana-ref}/infrastructure-ui-settings-kb.html[Infrastructure app settings] in the {kibana-ref}/settings.html[Kibana configuration file].
+The default configuration uses the `metricbeat-*` index pattern to query the data.
+The default configuration also defines field settings for things like timestamps and container names.
+
+If your metrics have custom index patterns, or use non-default field settings, you can override the default settings.
+Click *Configuration* to change the settings.
+This opens the *Configure source* fly-out dialog.
```

Comment:
This is no longer a flyout, but a Settings tab, see https://github.com/elastic/kibana/pull/42243 . The screenshot in the PR shows the Logs app, but the same change has been made in master and 7.x for the Infrastructure apps.

The contents of the tab should be the same as the old flyout, I believe it's just the name and how to access it that has changed.

Reply:
@skh Thanks for reviewing this. Yes, I was aware of the new settings tab, thanks, but it's not in the build I'm using yet. It's in my list of things to raise tickets for.